terraform/stacks/data_archive/byob_ica_v2.tf

```diff
@@ -1,41 +1,45 @@
 ################################################################################
-# Variables
-
-variable "analysis_data_bucket" {
-  description = "This bucket is holding production data."
-  default     = "org.umccr.data.analysis-data"
-}
-variable "analysis_data_prefix" {
-  description = "The prefix under which all analysis data is collected."
-  default     = "analysis_data/"
+# Local constants
+
+locals {
+  # The bucket holding all "production" data
+  production_data_bucket = "org.umccr.data.production"
+  # prefix used for the ICAv2 BYOB config (mapped to and accessible from ICA)
+  ica_byob_prefix = "icav2/byob/"
```

Comment:
Since we agreed to go with the one bucket approach, I guess this prefix does not make a lot of sense any more.
We could keep the storage separated between execution engines, but that goes a little against the "all active production data in one place" philosophy (and may again create access/sharing complications).

We could simply expose the whole bucket. Not sure if we'd be limiting our options... 
Thoughts? 
@alexiswl @victorskl

Reply:
A quick call, pls?

Reply:
> We could keep the storage separated between execution engines

Happy for them to share, I think it would be a nice-to-have if we could make sure all workflow execution points can output data in a similar format (so it'd essentially be impossible to tell which WES service created a directory).